### PR TITLE
[ENH] collapse_levels: multilevel index + find_replace on multiple columns 

### PR DESCRIPTION
**Pull request by mdini**
This PR resolves #(put issue number here).

# PR Checklist

Please ensure that you have done the following:

1. [x] PR in from a fork off your branch. Do not PR from <your_username>:master, but rather from <your_username>:<branch_name>.
2. [x] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

- [x] Run the command `make check` from pyjanitor's top-level directory. This will automatically run:
    - black formatting
    - pycodestyle checkin
    - running the test suite
    - docs build

## Code Changes

If you are adding code changes, please ensure the following:

- [x] Ensure that you have added tests.
- [x] Run all tests (`$ pytest .`) locally on your machine.
    - [x] Check to ensure that test coverage covers the lines of code that you have added.
    - [x] Ensure that all tests pass.

## Documentation Changes

If you are adding documentation changes, please ensure the following:

- [ ] Build the docs locally.
- [ ] View the docs to check that it renders correctly.

# PR Description

Please describe the changes proposed in the pull request:

- 
- 
- 

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

File: janitor/functions.py [hunk 1]
```diff
@@ -2054,19 +2054,21 @@ def collapse_levels(df: pd.DataFrame, sep: str = "_"):
...
     :param df: A pandas DataFrame.
     :param sep: String separator used to join the column level names
+    :param axis: 0 for index and 1 for columns 
     :returns: df
     """
 
     check("sep", sep, [str])
-
     # if already single-level, just return the DataFrame
+    df= df.transpose() if axis==0 else df
```
Comment (ericmjl) [inline janitor/functions.py]: @mdini code style suggestion here:

```suggestion
    df = df.transpose() if axis==0 else df
```

File: janitor/functions.py [hunk 2]
```diff
@@ -2054,19 +2054,21 @@ def collapse_levels(df: pd.DataFrame, sep: str = "_"):
...
     if not isinstance(df.columns.values[0], tuple):
         return df
 
     df.columns = [
         sep.join([str(el) for el in tup if str(el) != ""])
         for tup in df.columns.values
     ]
+    df= df.transpose() if axis==0 else df
```
Comment (ericmjl) [inline janitor/functions.py]: Same code style suggestion here.

```suggestion
    df = df.transpose() if axis==0 else df
```

**Comment by ericmjl** [comment]
@mdini would you do one final black and push please? I _think_ that should be it :smile:.

**Comment by mdini** [comment]
@ericmjl sorry i got issues with my virtual environment. Could you please tell me which version of python are you using? and which version of the package coverage. Thanks.

**Comment by ericmjl** [comment]
@mdini I'm using Python 3.6 and 3.7 on the build, and coverage - just use the latest and that should be ok.

**Comment by ericmjl** [comment]
@mdini giving you a heads-up here, I did some merge conflict resolution here.

**Comment by ericmjl** [comment]
@mdini, I think the final part here should be to pull in changes from `dev` one more time, push up here again, and then await CI passing!

**Comment by ericmjl** [comment]
@mdini there’s been a bunch of changes since PyCon. If you’d like to continue with the PR, please let me know!

**Comment by mdini** [comment]
@ericmjl sorry i was planning to work on this issue but couldnt afford the time. Please feel free to assign this issue to other contributors.

**Pull request closed without merging**